pkg/common/parameters.go

```diff
@@ -135,6 +140,8 @@ func ExtractAndDefaultParameters(parameters map[string]string, driverName string
 			for labelKey, labelValue := range paramLabels {
 				p.Labels[labelKey] = labelValue
 			}
+		case ParameterKeyProvisionedIOPSOnCreate:
+			p.ProvisionedIOPSOnCreate = v
```

Comment:
let's validate that this is an int64---see ConvertLabelsStringToMap, which does things like validate the characters used in the label. 

I think it would be best to validate the parameters as early as possible.

Reply:
Should we only validate int64 instead of also verifying the range? 

If we validate the range it means the range change in the future, we need to modify the validation logic each time and go through deployments.